Code/Framework/AzCore/AzCore/Component/TransformBus.h

```diff
@@ -286,6 +293,9 @@ namespace AZ
         //! A static transform is unmovable and does not respond to requests that would move it.
         virtual void SetIsStaticTransform([[maybe_unused]] bool isStatic) {}
         //! @}
+
+        //! Set the behavior at runtime when this entity's parent's transform changes.
+        virtual void SetOnParentChangedBehavior([[maybe_unused]] OnParentChangedBehavior onParentChangedBehavior) {}
```

Comment:
Worth adding a getter to the API as well?

Reply:
Yeah, I guess it at least won't do any harm.